Sources/NIOPosix/MultiThreadedEventLoopGroup.swift

```diff
@@ -76,6 +76,7 @@ public final class MultiThreadedEventLoopGroup: EventLoopGroup {
                                    canEventLoopBeShutdownIndividually: Bool,
                                    selectorFactory: @escaping () throws -> NIOPosix.Selector<NIORegistration>,
                                    initializer: @escaping ThreadInitializer,
+                                   delegate: EventLoopDelegate?,
```

Comment:
this this should be `metricsDelegate` or similar

Reply:
What's also odd is that we seem to have one delegate which gets metrics from multiple EventLoops in that EventLoopGroup.

To make this useful to users we should aggregate the metrics from all loops in a group, no? If we don't give the user to stats from all loops collectively, then the user would have to perform the aggregation themselves -- likely with locks across all loops -- which isn't ideal.